### ISSUE TITLE
jump back from definition

### ISSUE THREAD

**Issue by thunderkid** [adobe/brackets#11755]
When you jump to a definition with ctrl+J, is there a way to jump back again? Other editors let you move backwards and forwards but I can't see such commands listed in brackets.

**Comment by Denisov21** [comment]
Great idea! I up-vote it!

**Comment by busykai** [comment]
Did you try [this extension](https://github.com/opensourceportfolio/brackets-navigation-history)? By the looks of it, it should accomplish what's needed.

**Comment by thunderkid** [comment]
Yes, it seems like it would. I didn't actually look for an extension - I use that feature so much personally that I'd just assumed it would be part of the core functionality. I've just started using sublime, which has it built in.

**Comment by busykai** [comment]
So does `vim` and many other code editors. We would normally monitor feature requests, if there's a popular extension which implements it, we ask the author to contribute it to the core. Many features landed this way. Meanwhile, you could use the extension and upvote [this Trello card](https://trello.com/c/0bIXHmog/1047-jump-to-code-block-start-end-matching-braces-select-block). It would help landing it sooner.

**Comment by abose** [comment]
https://github.com/opensourceportfolio/brackets-navigation-history/issues/2 logged with the extension repo- requesting for contribution to brackets core.
